doc/NEWS

```diff
@@ -560,3 +560,6 @@ Major changes from 1.8.0-jumbo-1 (December 2014) to 1.9.0-jumbo-1 (May 2019):
   mostly part of our development process and not the release, although some
   CI-related files do exist in our released tree.
   [Claudio, magnum; 2015-2019]
+
+- Addition of mosquitto2john.py for cracking of Eclipse Mosquitto passwd files.
+  [Blackfell; 2021]
```

Comment:
The `NEWS` addition should be to the proper section of that file, not to the end of file. The way it currently is makes the change appear to be part of 1.9.0-jumbo-1.

Reply:
Understood, I'll get to grips with the News file and sort that out, thanks!